verify.go

```diff
@@ -116,25 +116,60 @@ func (v *verifier) stop() {
 	v.stopCh <- struct{}{}
 }
 
-func (v *verifier) verifyImages(ctx context.Context, images []string) ([]byte, error) {
-	digests := map[string]string{}
-	for _, image := range images {
-		digest, err := v.verifyImage(ctx, image)
+func (v *verifier) verifyImages(ctx context.Context, images *RequestData) ([]byte, error) {
+	response := ResponseData{
+		Allow:   false,
+		Results: make([]Result, 0),
+	}
+
+	for _, image := range images.Containers {
+		digest, err := v.verifyImage(ctx, image.String())
 		if err != nil {
 			v.logger.Errorf("verification failed for image %s: %v", image, err)
 			return nil, errors.Wrapf(err, "failed to verify image %s", image)
 		}
 
-		digests[image] = digest
+		image.Digest = digest
+		response.Results = append(response.Results, Result{
+			Name:  image.Name,
+			Path:  image.Pointer,
+			Image: image.String(),
+		})
 	}
 
-	results := map[string]interface{}{}
-	results["verified"] = true
-	results["digests"] = digests
+	for _, image := range images.InitContainers {
+		digest, err := v.verifyImage(ctx, image.String())
+		if err != nil {
+			v.logger.Errorf("verification failed for image %s: %v", image, err)
+			return nil, errors.Wrapf(err, "failed to verify image %s", image)
+		}
+
+		image.Digest = digest
+		response.Results = append(response.Results, Result{
+			Name:  image.Name,
+			Path:  image.Pointer,
+			Image: image.String(),
+		})
+	}
+
+	for _, image := range images.EphemeralContainers {
+		digest, err := v.verifyImage(ctx, image.String())
+		if err != nil {
+			v.logger.Errorf("verification failed for image %s: %v", image, err)
+			return nil, errors.Wrapf(err, "failed to verify image %s", image)
+		}
+
+		image.Digest = digest
+		response.Results = append(response.Results, Result{
+			Name:  image.Name,
+			Path:  image.Pointer,
+			Image: image.String(),
+		})
+	}
```

Comment:
Can we add a log stating "verified %d containers %d initContainers %d ephemeralContainers" to help troubleshoot?